--- a/openwrt/qca/src/qca-legacy-uboot/include/956x.h
+++ b/openwrt/qca/src/qca-legacy-uboot/include/956x.h
@@ -3649,7 +3649,7 @@
 
 #define CPU_DDR_SYNC_MODE                   DDR_CTL_CONFIG_CPU_DDR_SYNC_SET(0)
 
-#define CPU_PLL_CONFIG1_NINT_VAL            CPU_PLL_CONFIG1_NINT_SET(0x1f)
+#define CPU_PLL_CONFIG1_NINT_VAL            CPU_PLL_CONFIG1_NINT_SET(0x2a) //Overclock CPU to 1050MHZ
 #define CPU_PLL_CONFIG_REF_DIV_VAL          CPU_PLL_CONFIG_REFDIV_SET(0x1)
 #define CPU_PLL_CONFIG_RANGE_VAL            CPU_PLL_CONFIG_RANGE_SET(0)
 #define CPU_PLL_CONFIG_OUT_DIV_VAL1         CPU_PLL_CONFIG_OUTDIV_SET(0)
@@ -3667,7 +3667,7 @@
 #define DDR_PLL_CONFIG_OUT_DIV_VAL1         DDR_PLL_CONFIG_OUTDIV_SET(0)
 #define DDR_PLL_CONFIG_OUT_DIV_VAL2         DDR_PLL_CONFIG_OUTDIV_SET(0)
 #define DDR_PLL_DITHER1_VAL                 DDR_PLL_DITHER1_DITHER_EN_SET(0) | \
-                                            DDR_PLL_DITHER1_NFRAC_MIN_SET(0) | \
+                                            DDR_PLL_DITHER1_NFRAC_MIN_SET(0x28f56) | \ //Overclock RAM to 666MHZ
                                             DDR_PLL_DITHER1_NFRAC_STEP_SET(0) | \
                                             DDR_PLL_DITHER1_UPDATE_COUNT_SET(0x0)
 

